packages/client/src/i18n/reducer.ts

```diff
@@ -87,32 +83,6 @@ const formatLocationLanguageState = (
   return languages
 }
 
-const formatRoleLanguageState = (
```

Comment:
These were caused by the difficulty of the previous way of defining the text labels in the database and not using message descriptor. A good example of why all type of formatting should be pushed to as close to rendering and as far away from the database as possible.